test/dlc_tests/sanity/test_pre_release.py

```diff
@@ -249,14 +249,20 @@ def _run_dependency_check_test(image, ec2_connection, processor):
     short_fw_version = re.search(r"(\d+\.\d+)", image).group(1)
 
     allow_openssl_cve_fw_versions = {
-        "tensorflow": ["1.15", "2.3", "2.4", "2.5", "2.6"],
-        "mxnet": ["1.9"],
-        "pytorch": [],
-        "huggingface_pytorch": ["1.8"],
-        "huggingface_tensorflow": ["2.4"]
+        "tensorflow": {
+            "1.15": ["cpu", "gpu", "neuron", "eia"],
+            "2.3": ["cpu", "gpu", "eia"],
```

Comment:
have you verified the openssl version in the EI/Neuron containers for these versions? This should be done prior to merge

Reply:
I've added exceptions for whatever was previously excepted. Will double check these to make sure.